### PR TITLE
Moving encryption_dummy to the apps repo

### PR DESCRIPTION
**Pull request by DeepDiver1975**
refs https://github.com/owncloud/core/issues/15950

@karlitschek @schiesbn

### CODE REVIEW

File: encryption_dummy/appinfo/app.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+$manager = \OC::$server->getEncryptionManager();
+$module = new \OCA\Encryption_Dummy\DummyModule();
```
Comment (nickvergessen) [inline encryption_dummy/appinfo/app.php]: this should be inside the closure, I failed when reworking that

File: encryption_dummy/appinfo/info.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<info>
+	<id>encryption_dummy</id>
+	<name>dummy encryption module</name>
+	<description>
+	This module does nothing, it is used for testing purpose only
+	</description>
+	<licence>AGPL</licence>
+	<author>Bjoern Schiessle</author>
+	<requiremin>8</requiremin>
```
Comment (nickvergessen) [inline encryption_dummy/appinfo/info.xml]: 8.1 :see_no_evil:

**Comment by karlitschek** [comment]
nice :+1:

**Comment by DeepDiver1975** [comment]
@nickvergessen feel free to push your changes in here - THX

File: encryption_dummy/appinfo/info.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+<info>
+	<id>encryption_dummy</id>
+	<name>dummy encryption module</name>
+	<description>
+	This module does nothing, it is used for testing purpose only
+	</description>
+	<licence>AGPL</licence>
+	<author>Bjoern Schiessle</author>
+	<requiremin>8.1</requiremin>
+	<shipped>false</shipped>
```
Comment (nickvergessen) [inline encryption_dummy/appinfo/info.xml]: Also changed this to false

**Comment by MorrisJobke** [comment]
:+1:

**Pull request merged**